Overview.src.html

```diff
@@ -2369,6 +2293,86 @@ <h4 id="should-response-to-request-be-blocked-due-to-nosniff?"><dfn title="shoul
 </ol>
 
 
+<h3>HTTP-redirect fetch</h3>
+
+<p class="note no-backref">This algorithm will be used by <cite>HTML</cite>'s "navigate" algorithm
+in addition to <span title=concept-http-fetch>HTTP fetch</span> above.
+<span data-anolis-ref>HTML</span>
+
+<p>To perform an <dfn title=concept-http-redirect-fetch>HTTP-redirect fetch</dfn> using
+<var>request</var> and <var>response</var>, with an optional <i>CORS flag</i>, run these steps:
+
+<ol>
+ <li><p>Let <var>actualResponse</var> be <var>response</var>, if <var>response</var> is not a
+ <span title=concept-filtered-response>filtered response</span>, and <var>response</var>'s
+ <span title=concept-internal-response>internal response</span> otherwise.
+
+ <li><p>Let <var>location</var> be the result of <span title=concept-header-parse>parsing</span>
+ `<code title>Location</code>` in <var>actualResponse</var>'s
+ <span title=concept-response-header-list>header list</span>.
+
+ <li><p>If <var>location</var> is null, return <var>response</var>.
+
+ <li><p>If <var>location</var> is failure, return a
+ <span title=concept-network-error>network error</span>.
+ <!-- only Gecko does this; and even that is currently more complicated -->
+
+ <li><p>Let <var>locationURL</var> be the result of
+ <span data-anolis-spec=url title=concept-url-parser>parsing</span> <var>location</var> with
+ <var>request</var>'s <span title=concept-request-current-url>current url</span>.
```

Comment:
```js
event.respondWith(fetch("//static.example.com/foo/", { redirect: 'manual' }))
```

Where:

* The request URL is `//example.com/bar/`
* Request has a manual redirect mode
* `//static.example.com/foo/` redirects to `/bar/`

This would result in redirecting to `//example.com/bar/` rather than `//static.example.com/bar/` right?